ang/test/civicase-base/factories/is-truthy.factory.spec.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-env jasmine */
+(() => {
+  describe('isTruthy', () => {
+    let isTruthy;
+
+    beforeEach(module('civicase-base'));
+
+    beforeEach(inject((_isTruthy_) => {
+      isTruthy = _isTruthy_;
+    }));
+
+    describe('when tested with "1"', () => {
+      it('returns true', () => {
+        expect(isTruthy('1')).toBe(true);
+      });
+    });
+
+    describe('when tested with "0"', () => {
+      it('returns false', () => {
+        expect(isTruthy('0')).toBe(false);
+      });
+    });
+
+    describe('when tested with 1', () => {
+      it('returns true', () => {
+        expect(isTruthy(1)).toBe(true);
+      });
+    });
+
+    describe('when tested with 0', () => {
+      it('returns false', () => {
+        expect(isTruthy(0)).toBe(false);
+      });
+    });
+
+    describe('when tested with "true"', () => {
+      it('returns false', () => {
+        expect(isTruthy(true)).toBe(true);
+      });
+    });
+
+    describe('when tested with "false"', () => {
+      it('returns false', () => {
+        expect(isTruthy(false)).toBe(false);
+      });
+    });
+  });
```

Comment:
do we consider the strings "true" as true, and "false" as false? do we need to cover those?

Reply:
I dont think we need it, because that scenario does not happen for us. But in future, we can add if needed.